vnext/ReactCommon/ReactCommon.vcxproj

```diff
@@ -116,9 +116,6 @@
     <ClInclude Include="$(ReactNativeDir)\ReactCommon\cxxreact\RecoverableError.h" />
     <ClInclude Include="$(ReactNativeDir)\ReactCommon\cxxreact\SystraceSection.h" />
     <ClInclude Include="$(ReactNativeDir)\ReactCommon\jsi\JSCRuntime.h" />
-    <ClInclude Include="$(ReactNativeDir)\ReactCommon\jsi\V8Runtime.h" Condition="'$(Platform)' != 'arm' AND '$(USE_V8)' == 'true'" />
```

Comment:
>V8Runtime.h [](start = 58, length = 11)

We should probably remove these files if the functionality now moved into the external NuGet

Reply:
Yep.. these can be removed from our build scripts.. But, Android still builds these hence we can't delete the files ..